addons/interaction/functions/fnc_initAce3Interactions.sqf

```diff
@@ -0,0 +1,71 @@
+/**
+ * Inits ACE3 Interactions likes dragging, carrying and cargo.
+ * Workaround is needed for some classes, that can't use ACE3 config entries.
+ *
+ * Arguments:
+ * 0: Equipment <OBJECT>
+ * 1: Dragging Settings <ARRAY>
+ * 2: Carrying Settings <ARRAY>
+ * 3: Cargo Settings <ARRAY>
+ *
+ * Returns:
+ * none
+ */
+
+params["_equipment", "_aceDragging", "_aceCarrying", "_aceCargo", "_interactionConditions"];
+
+// this function only runs once on mission start on server or on client if it's no multiplayer
+if(isServer || !isMultiplayer) then
```

Comment:
isServer is already true if called in singleplayer

Reply:
You are right. I will remove `|| !isMultiplayer`